scripts/consumer_apps_builder.py

```diff
@@ -105,9 +104,14 @@ def get_app_name_from_profile():
     return tree.getroot().get("name")
 
 
-def move_apk(app_id):
+def move_apk(app_id, build_type):
     subprocess.call(["mkdir", "-p", "./build/outputs/consumer_apks"]) 
-    subprocess.call(["mv", "./build/outputs/apk/commcare-odk-standalone-debug.apk", "./build/outputs/consumer_apks/{}.apk".format(app_id)])
+    if build_type == 'd':
+        original_apk_filename = "./build/outputs/apk/commcare-odk-standalone-debug.apk"
+    else:
+        original_apk_filename = "./build/outputs/apk/commcare-odk-standalone-release.apk"
+    subprocess.call(["mv", original_apk_filename, "./build/outputs/consumer_apks/{}.apk".format(app_id)])
```

Comment:
Not specific to this PR, but before you finish up work on this feature I would try to use specific file util libs to move/copy/make dirs. I think `shutil` is a good one